### ISSUE TITLE
Wakatime Does not start up on insiders.vscode.dev

### ISSUE THREAD

**Issue by Lukas-Batema** [wakatime/vscode-wakatime#269]
# Error
```bash
WARN Via 'product.json#extensionEnabledApiProposals' extension 'ms-dotnettools.dotnet-interactive-vscode' wants API proposal 'notebookConcatTextDocument' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned. log.ts:307:11
[Extension Host] [WakaTime][DEBUG] Initializing WakaTime v18.1.0 console.ts:137:34
Activating extension 'WakaTime.vscode-wakatime' failed: window is not defined. mainThreadExtensionService.ts:110:10
    $onExtensionActivationError mainThreadExtensionService.ts:110
    _doInvokeHandler rpcProtocol.ts:473
    _invokeHandler rpcProtocol.ts:458
    _receiveRequest rpcProtocol.ts:374
    _receiveOneMessage rpcProtocol.ts:296
    S rpcProtocol.ts:161
    invoke event.ts:575
    deliver event.ts:779
    fire event.ts:740
    onmessage webWorkerExtensionHost.ts:209
Source map error: Error: request failed with status 404
Resource URL: https://wakatime.vscode-unpkg.net/WakaTime/vscode-wakatime/18.1.0/extension/dist/web/extension.js#vscode-extension
Source Map URL: extension.js.map
```

**Comment by alanhamlett** [comment]
What version of insiders are you using? It shouldn't be trying to load the `web` extension unless it's running in a browser.

**Comment by Lukas-Batema** [comment]
It is running in a browser, I am using https://insiders.vscode.dev...

**Comment by alanhamlett** [comment]
Fixed in v18.1.1.

**Comment by Lukas-Batema** [comment]
Thank you :)
